Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarksViewController.swift

```diff
@@ -690,3 +691,18 @@ extension BookmarksViewController {
         }
     }
 }
+
+extension BookmarksViewController: PanModalPresentable {
+    var panScrollable: UIScrollView? {
+        (parent as? PanModalPresentable)?.panScrollable
+    }
+
+    func shouldRespond(to panModalGestureRecognizer: UIPanGestureRecognizer) -> Bool {
+        // This prevents 2 problems:
+        // 1. Closing the pan modal when user swipes down on bookmarks list
```

Comment:
Why would we want to disable them dismissing the pan modal from here?

Reply:
if you have 30+ bookmarks, scroll to the bottom, then when you scroll up it dismissed the view

It could be because the bookmarks screen is pushed on its own navigation screen, i did notice this problem in other screens, check it out

Reply:
Its likely just a side-effect of flicking upwards to the top of the page with enough velocity that would cause it to start dismissing, but I dont think we should disable the gesture entirely for the page. Likely just needs adjusting to allow bounce if the scroll began below, though I'm not sure how much work it is to adjust PanModal to do so